blog.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="description" content="Adopt a dog in Seattle">
+    <meta name="viewpoint" content="width=device-width, initial-scale=1">
+    <title>Blog</title>
+    <link href="https://fonts.googleapis.com/css?family=Bitter|Indie+Flower|Muli:400,800" rel="stylesheet">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+
+  <header>
+    <div class="header-flex">
+      <div class="header-box">
+        <a href="#main-content" class="hidden">Skip to main content</a><br>
+        <img class="header-img" src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+        <h2>Seattle's premiere dog adoption agency</h2>
+      </div>
+      <div class="header-box">
+        <p class="light"><i class="fas fa-shopping-cart"></i>&nbsp;<strong>$1,234.56</strong></p>
```

Comment:
There is an included shopping cart svg, perhaps you could try to include that rather than pulling down font awesome?